### PR TITLE
Azure disclaimer

### PR DESCRIPTION
**Pull request by datosh**
### Proposed change(s)

- add disclaimer for potentially long inits on Azure
- add @flxflx as codeowner for docs (you were not automatically added as reviewer, let's see if it works once it landed in `main`)

### CODE REVIEW

**Review by flxflx — APPROVED** [review]

File: docs/docs/getting-started/first-steps.md [hunk 1]
```diff
@@ -128,6 +128,12 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
 4. Initialize the cluster
 
+    :::caution
+
+    Initialization on **Azure** might be slow and might take up to 60 minutes to initialize all K8s nodes. We are already working with Microsoft to resolve the issue. The stability of already initialized clusters is not affected.
```
Comment (katexochen) [inline docs/docs/getting-started/first-steps.md:133]:
```suggestion
    Initialization on **Azure** might be slow and might take up to 60 minutes to initialize all Kubernetes nodes. We're already working with Microsoft to resolve the issue. The stability of already initialized clusters isn't affected.
```
Imo we shouldn't use the 'k8s' short form in our docs. And is there any reason to ignore the vale language recommendations?

Comment (flxflx) [reply docs/docs/getting-started/first-steps.md:133]: Agreed. Shouldn't use K8s.

Comment (flxflx) [reply docs/docs/getting-started/first-steps.md:133]: Also agree to that we should follow Vale recommendations except for the "we", which is OK in most cases.

Comment (datosh) [reply docs/docs/getting-started/first-steps.md:133]: I've added #820 to fix those.

**Pull request merged**